docs/configuration/OVERVIEW.md

```diff
@@ -86,50 +87,86 @@ components at their instantiation time
 
 ### Dynamic configuration
 
-- The default configuration is extracted by __cms-adapters__ tool and bundled in a JSON file will be exposed in the CMS
+- The default configuration is extracted by __@o3r/components:extractor__ and bundled in a JSON file which will be exposed in the CMS
 
-- In the CMS plugin the Business Analyst will modify configuration of components and this ones will be exposed (in the
+- In the CMS plugin the Business Analyst will modify the configuration of components which will be exposed (in the
   server) as dynamic content of the app
 
-- The application will do an HTTP call, get the custom config and inject it in the store.
+- The application will do an HTTP call to get the custom config and inject it in the store.
 
-__No need of rebuild/redeploy the app__
+[!NOTE]
+> There is no need to rebuild/redeploy the application to apply these changes.
 
 ## Config types lib/app
 
 - At __library__ level we have the config for components (__blocks, components, elements__).
 
-A block component class should implement __Block__ interface from __@o3r/core__ to be identified by cms adapter
+A block component class should specify __Block__ for __componentType__ in the __O3rComponent__ to be identified by the extractor.
+```typescript
+@O3rComponent({
+  componentType: 'Block'
+})
+@Component({
+  selector: 'o3r-block',
+  template: ''
+})
+export class BlockComponent {}
+```
 
 - At application level:
 
-__Page config__ - each page component class should implement __Page__ interface available in __@o3r/core__ in order to
-be identified by cms adapter
+__Page config__ - each page component class should specify __Block__ for __componentType__ in the __O3rComponent__ to be identified by the extractor.
+```typescript
+@O3rComponent({
+  componentType: 'Page'
+})
+@Component({
+  selector: 'o3r-page',
+  template: ''
+})
+export class PageComponent {}
+```
 
 __Application config__
-The application can have 2 types of configs depending on the use cases: __pre-bootstrap__ and __runtime (
-post-bootstrap)__ config
+The application can have 2 types of configuration depending on the use cases: __pre-bootstrap__ and __runtime (post-bootstrap)__.
 
 Pre-bootstrap config
 
-- Defined in one interface which will extend __AppBuildConfiguration__ interface available in __@o3r/core__ in order
-  to be identified by cms adapter
-- PreBootstrap config object is used for configurations needed before loading the angular app component, and it will be
-  injected by the cms in the index.html body tag data-bootstrapconfig as a data attribute
+- Defined in one interface extending the __AppBuildConfiguration__ from __@o3r/core__ in order
+  to be identified by the extractor
+- PreBootstrap config object is used for configurations needed before loading the Angular app component, and it will be
+  injected by the CMS in the index.html body tag data-bootstrapconfig as a data attribute
```

Comment:
It's not injected by the CMS, there is no runtime interactions with the CMS, it's used as an edition panel for the customization and the files are exported in the dynamic content folder afterwards. We should advertise that this part is linked to a piece of code server side that updates the index.html and what requires to be added with an example